scheduler/src/cook/mesos/api.clj

```diff
@@ -1172,9 +1177,9 @@
      ;; Implementing put! doesn't mean that PUT requests are actually supported
      ;; (see :allowed-methods above). It is simply what liberator eventually calls to persist
      ;; resource changes when conflict? has returned false.
-     :put! (partial create-jobs! conn)
+     :put! (partial create-jobs! conn force-container-user?)
```

Comment:
`force-container-user?` is passed as the *second* argument to `create-jobs!` here.